depends/check-ps3dev.sh

```diff
@@ -1,14 +1,16 @@
 #!/bin/sh
+#
 # check-ps3dev.sh by Naomi Peori (naomi@peori.ca)
+# Modified by luizfernandonb (luizfernando.nb@outlook.com)
 
 ## Check if $PS3DEV is set.
 if test ! $PS3DEV; then { echo "ERROR: Set \$PS3DEV before continuing."; exit 1; } fi
 
 ## Check for the $PS3DEV directory.
-( ls -ld $PS3DEV || mkdir -p $PS3DEV ) 1>/dev/null 2>&1 || { echo "ERROR: Create $PS3DEV before continuing."; exit 1; }
+( ls -ld $PS3DEV || mkdir -p $PS3DEV ) 1>/dev/null 2>&1 || { sudo mkdir $PS3DEV && sudo chown $USER:$USER $PS3DEV; } # { echo "ERROR: Create $PS3DEV before continuing."; exit 1; }
```

Comment:
Eh, no.  _Never_ use sudo in scripts.

Reply:
Why not?

Reply:
Privilege separation exists for a reason.  Escalation of privilege must always be a conscious decision by the operator.